src/App.js

```diff
@@ -1,26 +1,75 @@
 import React from 'react';
-import logo from './logo.svg';
+//import logo from './logo.svg';
+import 'bulma/css/bulma.css';
 import './App.css';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFood';
+import TodaysFood from './components/TodaysFood'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+
+class App extends React.Component {
+
+  state = {
+    foodList: foods,
+    search: false
+  }
+  handleAddFood = (event) =>{
+    event.preventDefault();
+    let inputs = event.currentTarget
+    this.setState({
+      foodList: [{name: inputs.name.value, calories: inputs.calories.value, image: inputs.image.value, quantity:0}, ...this.state.foodList],
+      search: false
+    })
+  }
+  handleSearch = (e) =>{
+    let currentSearch = e.currentTarget.value;
+    let filtered = foods.filter((item)=>{
+      return item.name.toUpperCase().match(currentSearch.toUpperCase());
+    })
+    this.setState({
+      foodList: filtered
+    })
+  }
+  handleAddFoodToday = (id, q) =>{
+    let toModify = [...this.state.foodList];
+    toModify[id].quantity = q;
+    this.setState({
+      foodList: toModify
+    })
+  }
+  handleDelete = (id) =>{
+    let toDelete = [...this.state.foodList];
+    toDelete.splice(id,1);
+    this.setState({
+      foodList: toDelete
+    })
+  }
+  render(){
+    return (<div style = {{display: 'flex',flexDirection: 'column', alignItems:'space-between'}}>
```

Comment:
for best practices, start your JSX files in a new line below the return, also watch for indentation as I see different ones used in this JSX. Use the commands for auto indentation to see the difference.